jwql/example_config.json

```diff
@@ -2,21 +2,31 @@
     "admin_account" : "",
     "auth_mast" : "",
     "connection_string" : "",
-    "database" : {
+    "databases" : {
```

Comment:
Why is database[s] needed at all anymore? Isn't everything now nested under "django_databases"?

Reply:
The main reason is that all of the monitors are still using the existing methods of accessing the databases, and there’s no reason to remove that interface until the clean up when we also remove the sqlalchemy code

Reply:
perfect, thanks!